framework/encode/custom_vulkan_struct_handle_wrappers.cpp

```diff
@@ -124,5 +124,31 @@ void UnwrapStructHandles(VkWriteDescriptorSet* value, HandleUnwrapMemory* unwrap
     }
 }
 
+const void* TrackPNextStruct(const void* value, HandleUnwrapMemory* unwrap_memory)
+{
+    const void*        ret        = nullptr;
+    VkBaseInStructure* pnext      = nullptr;
+    VkBaseInStructure* last_pnext = nullptr;
+    auto               base       = reinterpret_cast<const VkBaseInStructure*>(value);
+    bool               first      = true;
+    while (base != nullptr)
+    {
+        // Use CopyPNextStruct, instead of UnwrapPNextStructHandles because it doesn't need to wrap handles.
+        pnext = CopyPNextStruct(base, unwrap_memory);
+        if (first == true)
```

Comment:
`if (first)`

Reply:
Fixed. Removed `first` and used `if (ret == nullptr)`